mino/minogrpc/server.go

```diff
@@ -742,3 +770,20 @@ func uriFromContext(ctx context.Context) string {
 
 	return apiURI[0]
 }
+
+func clientTracingDecorator(ctx context.Context, span opentracing.Span, method string,
```

Comment:
I would add a quick comment. Also a function's name should describe an action with a verb. Maybe `decorateClientTrace` ?